src/utils.js

```diff
@@ -560,4 +560,35 @@ Runs common initialize code on the svg before the chart builds
 */
 nv.utils.initSVG = function(svg) {
     svg.classed({'nvd3-svg':true});
-};
\ No newline at end of file
+};
+
+
+/*
+Sanitize and provide default for the container height.
+*/
+nv.utils.sanitizeHeight = function(height, container) {
+    return (height || parseInt(container.style('height')) || 400);
+}
+
+
+/*
+Sanitize and provide default for the container width.
+*/
+nv.utils.sanitizeWidth = function(width, container) {
+    return (width || parseInt(container.style('width')) || 960);
```

Comment:
Yes, this will allow the angularjs-nvd3-directives project to use sanitizeWidth and santizeHeight to calculate the proper view box values instead of getting "0 0 undefined undefined".